### ISSUE TITLE
Supporting contracts with no trump specified as just 'N' rather than 'NT'

### ISSUE THREAD

**Issue by badderstuber** [richardschneider/pbn#8]
I'm seeing a a lot of PBN's that have contracts such as 3N rather than 3NT (not sure where they are being generated). Would it be possible to support that contract definition even though it's clearly written in the spec that it should be NT. I understand if you don't want to support this, but if you do I'll happily make a pull request for the change. Thanks

**Comment by richardschneider** [comment]
Happy to add support for `N` as well as `NT`.

**Comment by badderstuber** [comment]
Thanks Richard. Any idea when this might get released?

**Comment by richardschneider** [comment]
It was released in [v1.6.0](https://github.com/richardschneider/pbn/releases/tag/v1.6.0).  If you cliick `Watch` at the top of the Github page, you will get an email.

Also, I practice CI/CD with semantic-release.  Once a PR is merged into `master`, a new release is automagically deployed to NPM.

**Comment by badderstuber** [comment]
ok great, thanks Richard, much appreciated.
